subprojects/logging/src/main/java/org/gradle/internal/deprecation/DeprecationLogger.java

```diff
@@ -31,17 +31,28 @@
  * The basic deprecation message structure is "Summary. DeprecationTimeline. Context. Advice. Documentation."
  *
  * The deprecateX methods in this class return a builder that guides creation of the deprecation message.
+ *
  * Summary is populated by the deprecateX methods in this class.
+ *
  * Context can be added in free text using {@link DeprecationMessageBuilder#withContext(String)}.
+ * It should provide any _instance specific_ details about the deprecated usage that would help a user identify
```

Comment:
Use `<em>` here for emphasis maybe?

Reply:
I think "instance" is not well-defined here. Do you mean the instance the deprecation is related to, or the instance of deprecation?

I think an example with tasks or a configuration could help clarify what you mean here.